common/src/main/java/dev/latvian/mods/kubejs/item/OutputItem.java

```diff
@@ -9,36 +9,48 @@
 import net.minecraft.world.item.ItemStack;
 
 public class OutputItem implements OutputReplacement {
-	public static final OutputItem EMPTY = new OutputItem(ItemStack.EMPTY, Double.NaN);
+	public static final OutputItem EMPTY = new OutputItem(ItemStack.EMPTY, Double.NaN, 0, 0);
 
-	public static OutputItem of(ItemStack item, double chance) {
-		return item.isEmpty() ? EMPTY : new OutputItem(item, chance);
+	public static OutputItem of(ItemStack item, double chance, int minRolls, int maxRolls) {
+		return item.isEmpty() ? EMPTY : new OutputItem(item, chance, minRolls, maxRolls);
 	}
 
 	public static OutputItem of(Object o) {
 		if (o instanceof OutputItem out) {
 			return out;
 		} else if (o instanceof ItemStack stack) {
-			return of(stack, Double.NaN);
+			return of(stack, Double.NaN, 0, 0);
 		}
 
-		return of(ItemStackJS.of(o), Double.NaN);
+		return of(ItemStackJS.of(o), Double.NaN, 0, 0);
 	}
 
 	public final ItemStack item;
 	public final double chance;
+	public final int minRolls;
+	public final int maxRolls;
 
-	protected OutputItem(ItemStack item, double chance) {
+	protected OutputItem(ItemStack item, double chance, int minRolls, int maxRolls) {
 		this.item = item;
 		this.chance = chance;
+		this.minRolls = minRolls;
+		this.maxRolls = maxRolls;
```

Comment:
If this is introduced, we will need to add a constructor with the old (ItemStack, double) signature to avoid API breakage.

Reply:
Or rather, added the old signature back